covidalert/src/App.js

```diff
@@ -55,6 +52,18 @@ export default class extends Component {
     await this.verify();
   };
 
+  setUpKeyPair = async () => {
+    let keyPair = await retrieveItem('keyPair');
+    if (keyPair) {
+      keyPair = JSON.parse(keyPair);
```

Comment:
This line needs to also parse the keys directly and create the instance. The current method doesn't attach the instance methods needed for the keys later on in the code.

Reply:
Thank you. I fixed this.